src/Tizen.NUI/src/public/Window.cs

```diff
@@ -1111,11 +1112,14 @@ public void SetAvailableOrientations(List<Window.WindowOrientation> orientations
             {
                 for (int i = 0; i < orientations.Count; i++)
                 {
-                    orientationArray.PushBack(new PropertyValue((int)orientations[i]));
+                    PropertyValue value = new PropertyValue((int)orientations[i]);
+                    orientationArray.PushBack(value);
```

Comment:
is the `PropertyValue value` copied into `orientationArray` when `PushBack()`?
if it is not copied surely, then the `value` is disposed immediately and the value will be deleted.
then `orientationArray` will not have any data.
could you confirm this?